README.adoc

```diff
@@ -384,7 +384,7 @@ Available recipes:
 
 The argument to `--list-heading` replaces both the heading and the newline
 following it, so it should contain a newline if non-empty. It works this way so
-you can suppress the heading line entirely by passing the empty string:
+you can suppress the heading line entirely by passing an empty string:
```

Comment:
"the empty string" sounds weird, but it's correct, so let's keep it. ("an empty string" is also correct, of course)

Reply:
Reverted